test/spec/helper.state.js

```diff
@@ -361,7 +361,7 @@ test('getConfigurationFromQueryString should parse page as number and be consist
 
   var queryString = helper.getStateAsQueryString({filters: filters});
 
-  var partialStateFromQueryString = algoliasearchHelper.AlgoliaSearchHelper.getConfigurationFromQueryString(
```

Comment:
Maybe we can add one so that no developer will remove it (BUT when we do a V3 for example).